docs/products/kafka/howto/prevent-full-disks.rst

```diff
@@ -0,0 +1,62 @@
+Prevent full disks
+===================
+
+To prevent **Aiven for Apache Kafka** from malfunctioning, the Aiven platform detects when the available disk space in **Aiven for Apache Kafka** services is running low and when the usage exceeds 90% notifies the customer.
+
+If any of the nodes in the service exceeds the critical threshold of disk usage (more than 97%), the access control list (ACL) used to authorize API requests by Apache Kafka clients is updated on all nodes to prevent operations that would increase disk usage. Such operations include:
+
+-  The ``Write`` and ``IdempotentWrite`` operations that clients use to produce new messages
+
+-  The ``CreateTopics`` operation that creates one or more topics, each carrying some overhead on disk
+
+When the ACL blocks write operations, clients see an error similar to the following example from the ``python-kafka`` client::
+
+   TopicAuthorizationFailedError: [Error 29] TopicAuthorizationFailedError: mytopic
+
+
+To ensure that enough disk space is available you can take one of the following actions:
+
+Upgrade to a larger service plan
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+#. Log in to the `Aiven web console <https://console.aiven.io/>`_ and select your service.
+
+#. On the *Overview* page, scroll down to *Service plan* and click **Upgrade plan** .
```

Comment:
Same question, not sure when to use bold text or italic.

Reply:
We have bold for interactive UI elements, italics for non-interactive UI elements.